src/baldr/nodeinfo.cc

```diff
@@ -10,7 +10,7 @@ using namespace valhalla::baldr;
 
 namespace {
 
-json::MapPtr access_json(uint16_t access) {
+json::MapPtr access_jsonNI(uint16_t access) {
```

Comment:
Very similar to implementation in https://github.com/valhalla/valhalla/blob/bea80dc49c7199472122dd275adf65de1b784232/src/baldr/directededge.cc#L9
Probably makes sense to extract somewhere for common use (but I don' know the right place)